Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/gradle:jdk19-jammy AS build
```

Comment:
Would it be possible to use the jdk20 to be in sync with the `develop` branch sdk?

Reply:
Yes :)